[CI] ForecastIT.testOverflowToDisk

**Build scan**: https://gradle-enterprise.elastic.co/s/dakbqxzmvfxnm 

**Repro line**: 
```
/gradlew ':x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner' --tests "org.elasticsearch.xpack.ml.integration.ForecastIT.testOverflowToDisk" -Dtests.seed=E001230D5CCE4F9 -Dtests.security.manager=true -Dtests.locale=sr-CS -Dtests.timezone=Pacific/Easter -Druntime.java=8 -Dtests.fips.enabled=true
```
**Reproduces locally?**: No

**Applicable branches**: 7.9 onwards

**Failure history**:

**Failure excerpt**:
```
java.lang.AssertionError: 
Expected: one of {<finished>}
     but: was <scheduled>
	at __randomizedtesting.SeedInfo.seed([E001230D5CCE4F9:41D0AC6ABCB2ED49]:0)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
	at org.junit.Assert.assertThat(Assert.java:956)
	at org.junit.Assert.assertThat(Assert.java:923)
	at org.elasticsearch.xpack.ml.integration.MlNativeAutodetectIntegTestCase.lambda$waitForecastStatus$1(MlNativeAutodetectIntegTestCase.java:301)
	at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:951)
	at org.elasticsearch.xpack.ml.integration.MlNativeAutodetectIntegTestCase.waitForecastStatus(MlNativeAutodetectIntegTestCase.java:298)
	at org.elasticsearch.xpack.ml.integration.MlNativeAutodetectIntegTestCase.waitForecastToFinish(MlNativeAutodetectIntegTestCase.java:292)
	at org.elasticsearch.xpack.ml.integration.ForecastIT.testOverflowToDisk(ForecastIT.java:244)
```

--- a/x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/MlNativeAutodetectIntegTestCase.java
+++ b/x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/MlNativeAutodetectIntegTestCase.java
@@ -299,7 +299,7 @@ abstract class MlNativeAutodetectIntegTestCase extends MlNativeIntegTestCase {
             ForecastRequestStats forecastRequestStats = getForecastStats(jobId, forecastId);
             assertThat(forecastRequestStats, is(notNullValue()));
             assertThat(forecastRequestStats.getStatus(), in(status));
-        }, 30, TimeUnit.SECONDS);
+        }, 60, TimeUnit.SECONDS);
     }
 
     protected void assertThatNumberOfAnnotationsIsEqualTo(int expectedNumberOfAnnotations) throws IOException {

